apps/fortuna/src/config.rs

```diff
@@ -158,3 +158,72 @@ pub struct EthereumConfig {
     /// The gas limit to use for entropy callback transactions.
     pub gas_limit: U256,
 }
+
+#[derive(Args, Clone, Debug)]
+#[command(next_help_heading = "Provider Config Options")]
+#[group(id = "ProviderConfig")]
+pub struct ProviderConfigOptions {
+    /// Path to a configuration file containing provider configuration for the list of supported blockchains
+    /// The commitments should be sorted in ascending order by their original_committed_sequence_number.
+    #[arg(long = "provider-config")]
+    #[arg(env = "FORTUNA_PROVIDER_CONFIG")]
+    #[arg(default_value = "provider-config.yaml")]
+    pub provider_config: String,
+}
+
+#[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
+pub struct ProviderConfig {
+    pub chains: HashMap<ChainId, ProviderChainConfig>,
+}
+
+impl ProviderConfig {
+    pub fn load(path: &str) -> Result<ProviderConfig> {
+        // Open and read the YAML file
+        let yaml_content = fs::read_to_string(path)?;
+        let config: ProviderConfig = serde_yaml::from_str(&yaml_content)?;
+        Ok(config)
+    }
+
+    /// Gets the provider chain config from the Hashmap. It will check that the commitments are in order.
+    /// If not, it will return an Err. Else, the config will be returned.
+    pub fn get_chain_config(&self, chain_id: &ChainId) -> Result<ProviderChainConfig> {
+        let provider_chain_config = self.chains.get(chain_id).map(|x| x.clone()).ok_or(anyhow!(
```

Comment:
I think it's more natural to have these configs in the chain config yaml file. We see all the configs in the same place we need less checks like this